.github/workflows/build-docker.yaml

```diff
@@ -0,0 +1,40 @@
+# Controls when the workflow will run
+on:
+  pull_request:
+    branches:
+      - 'main'
+
+permissions:
+  packages: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: docker/setup-qemu-action@v2
+      - id: buildx
+        uses:
+          docker/setup-buildx-action@v2
+      - #if: github.event_name != 'pull_request'
```

Comment:
is this comment intentional?

Reply:
Going to remove them needed it for testing